scripts/travis-ci.sh

```diff
@@ -14,7 +14,7 @@ if [ "$TRAVIS_OS_NAME" == "linux" ]; then
     # Which docker image to use to run the CI. Defaults to Docker Hub.
     # Overwrite with CI_IMAGE=docker/image/path environment variable.
     # Keep this in sync with .github/workflows/ci.yml.
-    : "${CI_IMAGE:=shiftcrypto/bitbox-wallet-app:11}"
+    : "${CI_IMAGE:=shiftcrypto/bitbox-wallet-app:12}"
```

Comment:
unfortunately, .github/workflows/ci.yml wasn't updated. i guess the comment on top doesn't help. so, github should still be building with the version 11, i assume.